### ISSUE TITLE
[v3.0.2] Default Character Sheet Doesn't Honor Modified system.attributes.hp.max

### ISSUE THREAD

**Issue by nullreturned** [foundryvtt/dnd5e#3054]
Foundry Version: v11.315
DnD System Version: 3.0.2
Modules required: Dynamic effects using Active Effects (DAE) v11.3.1

When using Effects and enabling an effect of "Add" against "system.attributes.hp.max", the "Default 5e Character Sheet" does not honor the effect or the modified HP Maximum.  When switching back to the "Legacy 5e Character Sheet", the HP information shown on the sheet honors the modified "system.attributes.hp.max".  This does require the module "Dynamic effects using Active Effects (DAE)" to give access to additional attributes that can be modified through effects.

Example of how I reproduced:
1.) Have a character and view hit points on the "Default 5e Character Sheet".  In my case, my character normally has a base of 96 / 96 hp
2.) Add a custom effect to them against "system.attributes.hp.max" with a function of "Add" and a value. In my case I used "36".
3.) Observe that the Details tab still shows the original values.  In my case it still shows 96 / 96 hp.
4.) Switch Character Sheet to "Legacy 5e Character Sheet".
5.) Observe that the character sheet now shows hp as <original value> / <modified value>.  In my case, it shows 96 / 132 hp.
6.) Update current HP to max.  In my case I set my current HP to 132 / 132 hp.
7.) Switch the the "Default 5e Character Sheet", and observe that HP shows original values.  In my case it still shows 96 / 96 hp.
8.) Switch the the "Legacy 5e Character Sheet", and observe that HP shows modified values.  In my case it now shows 96 / 132 hp.

From the looks of it, it's as if the "Default 5e Character Sheet" is not using any modified values per the Effects, even though the effect is active.  I have verified that this is not all effects, as other modifications (Upgrading Int Value with a Headband of Intellect, Saves modified with a Ring of Protection, etc) are working fine.

**Comment by nullreturned** [comment]
Also, an obvious quick fix is to use the Legacy Character sheet, but it would be nice to be able to use the new Default one.

**Comment by krbz999** [comment]
If you require a module to break something, then it is not an issue with the system.

**Comment by MaxPat931** [comment]
It is not recommended to alter `...hp.max` with an active effect, instead try using `...hp.tempmax`
https://github.com/foundryvtt/dnd5e/wiki/Active-Effect-Guide#hit-points

**Comment by nullreturned** [comment]
> If you require a module to break something, then it is not an issue with the system.

That is a valid point, but this worked perfectly fine with the system previously.  This is not a situation where everything was working, then I added the module and things broke.  We've been playing with the module for some time now and it "breaks" only on the upgrade with the character sheet.

**Comment by nullreturned** [comment]
> ct, instead try using `...hp.`

Thank you for pointing me to this guide.  When there was the v11 upgrade, we opted to recreate everything instead of continuing to use the original world.  This included recreating our characters which led to HP not being the same.  We used "" as the effect, but per the guide, we should have been using "system.attributes.hp.bonuses.overall".  Once I switch the effect to "system.attributes.hp.bonuses.overall", everything looks much better.

Thank you for providing me a reference that helped solve the issue.
